argoverse/utils/centerline_utils.py

```diff
@@ -119,6 +119,32 @@ def convert_lane_boundaries_to_polygon(right_lane_bounds: np.ndarray, left_lane_
     return polygon
 
 
+
+def convert_lane_boundaries3d_to_polygon3d(right_ln_bnd: np.ndarray, left_ln_bnd: np.ndarray) -> np.ndarray:
```

Comment:
@benjaminrwilson @wqi we already have a function named `convert_lane_boundaries_to_polygon()` right above this in the same file. It's intended for 2d, however, but do you think it makes sense to consolidate them? and make it more generic?